packages/utils/src/lang.js

```diff
@@ -10,10 +10,16 @@ export const isNonEmptyString = obj => Boolean(obj && isString(obj))
 
 export const isPureObject = obj => !Array.isArray(obj) && typeof obj === 'object'
 
+// Replace once ufo exports isRelative
 export const isUrl = function isUrl (url) {
   return ['http', '//'].some(str => url.startsWith(str))
 }
 
+// Replace once ufo exports isRelative
+export function isRelative (url) {
```

Comment:
We can support this in ufo apart from relative resolution support :)

Reply:
Linking: https://github.com/nuxt-contrib/ufo/pull/17

Reply:
+ ufo@0.6.9